Assets/Scripts/Buildings/Upgrades/HealthUpgrade.cs

```diff
@@ -1,25 +1,21 @@
-using UnityEngine;
-
 namespace Buildings.Upgrades
 {
     public class HealthUpgrade : UpgradeBase
     {
-        [SerializeField] private int[] health;
-
         protected override void Awake()
         {
             base.Awake();
-            if (Building.MaxLevel < health.Length)
-                Building.MaxLevel = health.Length;
+            if (Building.MaxLevel < upgradeValues.Length)
+                Building.MaxLevel = upgradeValues.Length;
         }
 
         protected override void Upgrade(int level)
         {
             // If upgrade doesn't exist, return
-            if (level > health.Length) return;
+            if (level > upgradeValues.Length) return;
```

Comment:
Same here: Würde es in die base schieben und dann hier nur `base.Upgrade(int level);`